packages/desktop/webpack.config.ts

```diff
@@ -189,16 +189,21 @@ const preloadPlugins = [
 ]
 
 const sentryPlugins = [
-    new SentryWebpackPlugin({
-        authToken: process.env.SENTRY_AUTH_TOKEN,
-        include: '.',
-        release: `${getAppName(prod)}@${NETWORK}-${version}`,
-        ignoreFile: '.sentrycliignore',
```

Comment:
do we want to stop using this file then? if so, lets remove it, or use it again instead of writing the ignored files directly in the file

Reply:
Looks like we can only use an array of files now, https://www.npmjs.com/package/@sentry/webpack-plugin#sourcemapsignore